core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart

```diff
@@ -246,6 +297,17 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
           
           resizeObserver.observe(document.body);
         }
+        
+        ${widget.scrollController != null ? '''
+          window.addEventListener('wheel', function (event) {
+            const deltaY = event.deltaY;
```

Comment:
What happens if user try to scroll horizontal? We will not handle this?

Reply:
If the user scrolls horizontally of course, `window.addEventListener('wheel')` will be triggered, depending on the deltaY value the scroll handler will handle, but it will always ensure consistent scrolling between the iframe and flutter's scroll view